includes/gateways/paypal/gateway-filters.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * PayPal Commerce Gateway Filters
+ *
+ * @package    easy-digital-downloads
+ * @subpackage Gateways\PayPal
+ * @copyright  Copyright (c) 2021, Sandhills Development, LLC
+ * @license    GPL2+
+ */
+
+namespace EDD\PayPal;
+
+/**
+ * Removes PayPal Standard from the list of available gateways while we're on the EDD Settings page.
+ * This prevents PayPal Standard from being enabled as a gateway if:
+ *
+ *      - The store owner has never used PayPal Standard; or
+ *      - The store Owner used PayPal Standard previously but has now been onboarded to PayPal Commerce.
+ *
+ * @param array $gateways
+ *
+ * @since 2.11
+ * @return array
+ */
+function maybe_remove_paypal_standard( $gateways ) {
+	if ( function_exists( 'edd_is_admin_page' ) && edd_is_admin_page( 'settings' ) && ! paypal_standard_enabled() ) {
```

Comment:
Since this is in EDD core, is the `function_exists` really necessary?

Reply:
Yes, because `edd_is_admin_page()` is only loaded in the admin area, but this filter we're hooking into can also run on the front-end.